security.rst

```diff
@@ -2431,6 +2431,18 @@ want to include extra details only for users that have a ``ROLE_SALES_ADMIN`` ro
           // ...
       }
 
+.. tip::
+
+    Using ``isGranted()`` checks authorization against the currently logged in user. If you need to check
+    against a user that is not the one logged in or if checking authorization when the user session is not
+    available in a CLI context (example: message queue, cronjob) ``userIsGranted()`` can be used to set the
```

Comment:
please note that method has been renamed to `isGrantedForUser()` in symfony/symfony#59214